### ISSUE TITLE
[BUG] - Appointment - Event Registration Date

### ISSUE THREAD

**Issue by rdblake21** [SFDO-Community-Sprints/Summit-Events-App#407]
**Describe the bug**
The Event Registration Date Formula field isn't surfacing any date.  It appears blank trying to reference "Event Registration Requested Date".

There is a thought that this field isn't being populated because the Registration Status defaults to "Registered" not "Requested". This is functionality that may have been stripped when refactoring happened to Summit Events.

There may still be value in keeping the fields, but at the Event level, one would need to determine whether "Registered" is the post "Started" status or "Requested", allowing the Event Manager to determine some of the functionality.

Otherwise, the forumla field needs to be updated to reference an appropriate Event Instance Date.

**To Reproduce**
1. Create a basic event
2. Create an Appointment Type and select it during registration.
3. Go to the Appointment Record associated to the registration.

**Expected behavior**
The date is populated based on the registration date of the instance.

**Screenshots**
![image](https://user-images.githubusercontent.com/60475518/169750628-0ba21927-454d-4c07-b519-a1cb7f27e93a.png)

**Comment by walt0019** [comment]
@rdblake21, at St. Thomas, we have populated that field if we are manually creating registration in SEA and the date might be different than the record Created Date, otherwise we are largely using the Created Date for reporting.

**Comment by tcdahlberg** [comment]
I have a fix in my branch were if the Event Registration Requested date is blank than use create date. Does that work for you @rdblake21 and @walt0019? The formula looks like this now:

```
IF( NOT( ISBLANK(Event_Registration__r.Event_Registration_Requested_Date__c)),

TEXT(MONTH(Event_Registration__r.Event_Registration_Requested_Date__c))+"/" +TEXT(DAY(Event_Registration__r.Event_Registration_Requested_Date__c))+"/" +TEXT(YEAR(Event_Registration__r.Event_Registration_Requested_Date__c)),

TEXT(MONTH(DATEVALUE(Event_Registration__r.CreatedDate)))+"/" +TEXT(DAY(DATEVALUE(Event_Registration__r.CreatedDate)))+"/" +TEXT(YEAR(DATEVALUE(Event_Registration__r.CreatedDate)))
 
)
```

**Comment by tcdahlberg** [comment]
Included in beta release 0.28.0.1

**Comment by tcdahlberg** [comment]
Included in production release 0.28.0.2
